composer.json

```
@@ -27,6 +27,7 @@
         "php-http/client-implementation": "^1.0 || ^2.0",
         "php-http/client-common": "^1.1.0 || ^2.0",
         "php-http/message": "^1.0 || ^2.0",
+	"php-http/message-factory": "^1.0",
```

Spaces instead of tabs:

```suggestion
        "php-http/message-factory": "^1.0",
```

---

thanks